tools/install_visual_studio_22.ps1

```diff
@@ -2,8 +2,9 @@
 
 New-Item -Path "C:\" -Name "src" -ItemType "directory" -Force
 
-Invoke-WebRequest -Uri https://aka.ms/vs/17/release/vs_professional.exe `
-    -OutFile C:\src\vs_professional.exe
+$VS_INSTALL_URL="https://download.visualstudio.microsoft.com/download/pr/d1ed8638-9e88-461e-92b7-4e29cc6172c3/30be9c2a5e40d5666eeae683e319b472c8bcc8231c7b346fe798f0ad0f7e498b/vs_Professional.exe"
```

Comment:
small nit : 
can you add a comment stating 1) the visual studio version number : it's hard to make it out from the link 2) maybe state that this should be in sync with the docker file url